### PR TITLE
Fix unicode error

### PR DESCRIPTION
**Pull request by lsetiawan**
This PR addresses #148, At least for the Envirodiy database https://github.com/ODM2/WOFpy/issues/148#issuecomment-312350361.

Reference: https://docs.python.org/2/library/stringio.html#module-cStringIO

cc. @miguelcleon

### CODE REVIEW

**Comment by emiliom** [comment]
Thanks! FYI: I probably won't review this until early next week! Ping me mid Monday if I haven't followed up.

File: wof/apps/spyned_1_1.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from __future__ import (absolute_import, division, print_function)
 
-import StringIO
+import cStringIO
```
Comment (ocefpaf) [inline wof/apps/spyned_1_1.py]: Why not `io.BytesIO` and make this a modern python code :smile:

**Comment by lsetiawan** [comment]
After further testing, this solution doesn't seem to work that well, and it is not robust enough. Closing this PR.

**Comment by ocefpaf** [comment]
@lsetiawan I have some experience with this (but the ascii-unicode-bytes is always a little bit painful.)

If you can create a few tests that shows the error I can send a PR once I get back from SciPy.

**Pull request closed without merging**